test/tools/test_driver.sh

```diff
@@ -54,6 +54,8 @@ module load nco
 module load ncl
```

Comment:
@slevisconsulting you should remove most of these changes. When you update to the latest version the py_env_create line is actually only done in one place. So the new conda name will also only be in one place.

Reply:
Ahhh, OK. Yes, actually this is done on line 384 for all machines, so there is no need for each machine. So yes, remove the lines you added such as 57 and 58 to run py_env_create and conda activate.